src/analyzers/analyzer.cpp

```diff
@@ -6,12 +6,7 @@
 #include "analyzers/filter_factory.h"
 #include "analyzers/multi_analyzer.h"
 #include "analyzers/token_stream.h"
-#include "analyzers/filters/alpha_filter.h"
-#include "analyzers/filters/empty_sentence_filter.h"
-#include "analyzers/filters/length_filter.h"
-#include "analyzers/filters/list_filter.h"
-#include "analyzers/filters/lowercase_filter.h"
-#include "analyzers/filters/porter2_stemmer.h"
+#include "analyzers/filters/all.h"
```

Comment:
Not using `all.h` here is a compilation time optimization (don't need to parse as many headers for this translation unit since we only use a few filters in it). Any reason to switch this?